### PR TITLE
[web] changing the integration tests to show errors

### PR DESCRIPTION
**Pull request by nturgut**
changing the integration tests to show errors instead of only showing 'pass' 'fail'.
Example stack trace
```
[+2643 ms] Failure Details:
[        ] Failure in method: Focused text field creates a native input element
[        ] ══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
[        ] The following TestFailure object was thrown running a test:
[        ]   Expected: 'New Value'
[        ]   Actual: 'New Valu'
[        ]    Which: is different. Both strings start the same, but the
[        ] actual value is missing the following trailing characters: e
[        ] When the exception was thrown, this was the stack:
[        ]     at Object.wrapException (http://localhost:51152/main.dart.js:3797:17)
[        ]     at Object.throwExpression (http://localhost:51152/main.dart.js:3810:15)
[        ]     at Object.fail (http://localhost:51152/main.dart.js:17257:16)
[        ]     at Object._expect (http://localhost:51152/main.dart.js:17254:9)
[        ]     at Object.expect0 (http://localhost:51152/main.dart.js:17230:9)
[        ]     at Object.expect (http://localhost:51152/main.dart.js:20901:9)
[        ]     at http://localhost:51152/main.dart.js:94294:17
[        ]     at _wrapJsFunctionForAsync_closure.$protected (http://localhost:51152/main.dart.js:6725:15)
[        ]     at _wrapJsFunctionForAsync_closure.call$2 (http://localhost:51152/main.dart.js:39680:12)
[        ]     at StackZoneSpecification__registerBinaryCallback__closure.call$0 (http://localhost:51152/main.dart.js:92295:21)
[        ] The test description was:
[        ]   Focused text field creates a native input element
[        ] ═════════════════════════════════════════════════════════════════
[        ] end of failure 1

```
The drive file is reduced to nothing now :)
```
import 'package:e2e/e2e_driver.dart' as e2e;

Future<void> main() async => e2e.main();
```

### CODE REVIEW

File: e2etests/web/regular_integration_tests/README.md [hunk 1]
```diff
@@ -5,12 +5,16 @@ configuration (e.g. PWA vs non-PWA packaging), please create another
 directory under e2etests/web. Otherwise tests such as text_editing, history,
 scrolling, pointer events... should all go under this package.
 
+Tests can be run on both 'release' and 'profile' modes. However 'release' mode
+will swallow the stack trace and only fails the test. Use 'profile' mode for
```
Comment (yjbanov) [inline e2etests/web/regular_integration_tests/README.md]: Is it actually swallowing the stack trace or is the stack trace obfuscated?

Comment (nturgut) [reply e2etests/web/regular_integration_tests/README.md]: I might have used the wrong word.

It only shows a small message, it does not show the detailed stack trace.

I traced it all the way here: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/assertions.dart#L501

Comment (yjbanov) [reply e2etests/web/regular_integration_tests/README.md]: Ah, I'd say "summarize" or "shorten" or something that does not imply that it swallows exceptions, which usually means catch the exception and not print any message at all.

File: e2etests/web/regular_integration_tests/README.md [hunk 2]
```diff
@@ -5,12 +5,16 @@ configuration (e.g. PWA vs non-PWA packaging), please create another
...
 # To run the application under test for traouble shooting purposes.
 flutter run -d web-server lib/text_editing_main.dart --local-engine=host_debug_unopt
 
 # To run the Text Editing test and use the developer tools in the browser.
-flutter run --target=test_driver/text_editing_e2e.dart -d web-server --web-port=8080 --release --local-engine=host_debug_unopt
+flutter run --target=test_driver/text_editing_e2e.dart -d web-server --web-port=8080 --profile --local-engine=host_debug_unopt
 
 # To test the Text Editing test with driver:
-flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --release --browser-name=chrome --local-engine=host_debug_unopt
+flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --profile --browser-name=chrome --local-engine=host_debug_unopt
```
Comment (ferhatb) [inline e2etests/web/regular_integration_tests/README.md]: why did you change to profile ?  I set it profile for treeshaking test to be able to parse main.dart.js

File: e2etests/web/regular_integration_tests/test_driver/image_loading_e2e_test.dart [hunk 3]
```diff
@@ -2,16 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:io';
+import 'package:e2e/e2e_driver.dart' as e2e;
```
Comment (ferhatb) [inline e2etests/web/regular_integration_tests/test_driver/image_loading_e2e_test.dart]: thanks for fixing!

File: e2etests/web/regular_integration_tests/README.md (hunk 2 above)
Comment (nturgut) [reply e2etests/web/regular_integration_tests/README.md]: The release more shortens the error stack trace: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/assertions.dart#L501

We can still put test in both release and profile modes.

I just wanted to say, if you have an error, profile mode shows the full stack trace. I'll change the wording :)

File: e2etests/web/regular_integration_tests/README.md (hunk 1 above)
Comment (nturgut) [reply e2etests/web/regular_integration_tests/README.md]: Sure, I'll change the wording :)

File: e2etests/web/regular_integration_tests/test_driver/image_loading_e2e_test.dart (hunk 3 above)
Comment (nturgut) [reply e2etests/web/regular_integration_tests/test_driver/image_loading_e2e_test.dart]: np!
